src/main/resources/templates/db_scripts/schema.json

```diff
@@ -64,7 +64,12 @@
       "run": "after",
       "snippet": "COPY po_line (id, jsonb) FROM 'data/poLines.data' ENCODING 'UTF8' DELIMITER E'\t';",
       "fromModuleVersion": "1.0"
-    }
+    },
+    {
+      "run": "after",
+      "snippet": "CREATE OR REPLACE VIEW receiving_history_view AS SELECT pieces.id AS id, json_build_object('id', pieces.jsonb->>'id', 'po_line_number', po_line.jsonb->>'po_line_number', 'title', po_line.jsonb->>'title', 'receiving_note', details.jsonb->>'receiving_note') AS jsonb FROM pieces LEFT OUTER JOIN po_line ON pieces.jsonb->>'id' = po_line.jsonb->>'id' LEFT OUTER JOIN details ON details.jsonb->>'po_line_id' = po_line.jsonb->>'id';",
```

Comment:
Shouldn't the remaining fields in the receiving history schema like caption, comment, supplement, po_line_id etc.. be handled here?

Reply:
Good catch, I have added all remaining fields.